src/python/pants/bsp/goal.py

```diff
@@ -29,17 +29,17 @@ def run(
         specs: Specs,
         union_membership: UnionMembership
     ) -> ExitCode:
-        scheduler_session = graph_session.scheduler_session.scheduler.new_session(
-            build_id="bsp", dynamic_ui=False
-        )
+        # scheduler_session = graph_session.scheduler_session.scheduler.new_session(
+        #     build_id="bsp", dynamic_ui=False
+        # )
```

Comment:
Usually we don't like keeping in commented out code. Because this is so experimental + being developed daily by you, not the end of the world, though.

Reply:
Will remove.